### PR TITLE
Unify usage of PCI and MDev prefixes

### PR DESCRIPTION
**Pull request by VirrageS**
**What this PR does / why we need it**: We have been using `PCI_RESOURCE`  and `MDEV_PCI_RESOURCE` constant in different places. This MR unifies usage.

**Special notes for your reviewer**: None

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -19,6 +19,9 @@ const (
 	HostRootMount                             = "/proc/1/root/"
 	CPUManagerOS3Path                         = HostRootMount + "var/lib/origin/openshift.local.volumes/cpu_manager_state"
 	CPUManagerPath                            = HostRootMount + "var/lib/kubelet/cpu_manager_state"
+
+	PCIResourcePrefix  = "PCI_RESOURCE"
+	MDevResourcePrefix = "MDEV_PCI_RESOURCE"
```
Comment (EdDev) [inline pkg/util/util.go:24]: Usually, if it is not clear where to place an entity, it ends up in some `util` package.
I do not think this is a good idea, it causes many parts of the codebase to depend on things that are irrelevant (e.g. all source code files that depend on this `util` got effected due to this change, but most have nothing to do with PCI or MDEV I suspect).

Comment (VirrageS) [reply pkg/util/util.go:24]: Do you have suggestion on where it should be moved then?

Comment (vladikr) [reply pkg/util/util.go:24]: @VirrageS I think staging/src/kubevirt.io/api/core/v1/types.go would be a better place for these contacts.

Comment (VirrageS) [reply pkg/util/util.go]: Thanks @vladikr. Moved the constants to `types.go`.

**Comment by vladikr** [comment]
Thanks @VirrageS 
Looks good to me.
/approve

@EdDev could you please take another look?

**Review by EdDev — APPROVED** [review]
Thank you!

**Pull request merged**